canonical/canonical.com
# feat: build /careers/hiring-process

## Done

- Built new page as per [doc]() and [design]()
- Added link to it in careers nav under "Working here"
- Added page link to sitemap
- Drive by:
  - bumped Vanilla version 

## QA

- View the site locally in your web browser at: http://0.0.0.0:8002/
- Check against doc and design
- Check that link is present in careers nav and routes to page as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14744
